workspaces/frontend/src/app/pages/WorkspaceKinds/summary/WorkspaceKindSummaryExpandableCard.tsx

```diff
@@ -65,7 +65,9 @@ const WorkspaceKindSummaryExpandableCard: React.FC<WorkspaceKindSummaryExpandabl
           <Flex wrap="wrap">
             <SectionFlex title="Total GPUs in use">
               <FlexItem>
-                <Content>
+                <Content
+                  style={{ fontSize: LargeFontSize.value, fontWeight: BoldFontWeight.value }}
```

Comment:
When https://github.com/kubeflow/notebooks/pull/476 is merged, we can apply the corresponding utility classes here:

```suggestion
                  className="pf-v6-u-font-size-4xl pf-v6-u-font-weight-bold"
```